File: src/main/java/com/example/mediaApp/model/domain/Address.java
package com.example.mediaApp.model.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Address {
    private Long id;
    private String country;
    private String city;
    private String zipCode;
    private String street;
    private String houseNum;
    private AppUser user;
}

File: src/main/angular-media-app/src/app/models/address.ts
export class Address {
    id: number;
    country: string;
    city: string;
    zipCode: string;
    street: string;
    houseNum: string;
    isEditing: boolean;

    constructor() {
        this.id = 0;
        this.country = "";
        this.city = "";
        this.zipCode = "";
        this.street = "";
        this.houseNum = "";
        this.isEditing = false;
    }

    
}
